enrichments/trace/internal/elastic/span.go

```diff
@@ -434,6 +462,43 @@ func (s *spanEventEnrichmentContext) enrich(
 	if cfg.ProcessorEvent.Enabled && s.exception {
 		se.Attributes().PutStr(AttributeProcessorEvent, "error")
 	}
+	if s.exceptionType == "" && s.exceptionMessage == "" {
+		// Span event does not represent an exception
+		se.Attributes().PutStr(AttributeEventKind, "event")
+		se.Attributes().PutStr(AttributeMessage, se.Name())
```

Comment:
I know that we're doing this in APM data, but I don't think we should set the event name as the message field.

Reply:
+1, I was quite confused with this. I will remove this.

Reply:
Removed in https://github.com/elastic/opentelemetry-lib/pull/99/commits/4361255189c752b09bf2bed60016060031fdf858